ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,156 @@
 <!--
-  -  NextCloud rules
-  -  Author: banditopazzo.
-  -  Updated by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
 <!--
-- It's needed to enable audit in Nextcloud if you want full log
-- It's needed to paste the following in /var/ossec/etc/ossec.conf for the decoder to work properly:
-<localfile>
-	<location>ABSOLUTE_PATH_TO_YOUR_NEXTCLOUD_.json_LOG_FILE</location>
-	<log_format>json</log_format>
-	<label key="@source">NextCloud</label>
-</localfile>
-- OPTIONAL
-<localfile>
-	<location>ABSOLUTE_PATH_TO_YOUR_AUDIT_.json_LOG_FILE</location>
-	<log_format>json</log_format>
-	<label key="@source">NextCloud</label>
-</localfile>
+  Enable audit in Nextcloud if you want full log
+  Paste the following in /var/ossec/etc/ossec.conf for the decoder to work properly:
+  <localfile>
+    <location>ABSOLUTE_PATH_TO_YOUR_NEXTCLOUD_.json_LOG_FILE</location>
+    <log_format>json</log_format>
+    <label key="@source">NextCloud</label>
+  </localfile>
+  OPTIONAL
+  <localfile>
+    <location>ABSOLUTE_PATH_TO_YOUR_AUDIT_.json_LOG_FILE</location>
+    <log_format>json</log_format>
+    <label key="@source">NextCloud</label>
+  </localfile>
  -->
 
 <!-- ID: 88200 - 88299 -->
```

Comment:
Move to header including product description like this sample

```
<!-- 
  Rules for:
    Apache HTTP Server 2.2 ID:    30100 - 30299
    Apache HTTP Server 2.4 ID:    30300 - 30499 
-->
```

Reply:
Done